common/src/main/java/net/william278/huskchat/util/PlaceholderReplacer.java

```diff
@@ -19,7 +20,9 @@ public static String replace(Player player, String message, HuskChat implementin
         placeholders.put("%suffix%", implementingPlugin.getDataGetter().getPlayerSuffix(player).isPresent() ? implementingPlugin.getDataGetter().getPlayerSuffix(player).get() : "");
         placeholders.put("%ping%", Integer.toString(player.getPing()));
         placeholders.put("%uuid%", player.getUuid().toString());
-        placeholders.put("%servername%", player.getServerName());
+        placeholders.put("%servername%", Settings.serverNameReplacement.getOrDefault(
+                        player.getServerName(), player.getServerName()
+                ));
```

Comment:
very minute style thing but would you mind cleaning up this multi-lining here before I merge?